src/utils/hints/tokensHints.js

```diff
@@ -1,5 +1,17 @@
 export const tokenHints = {
   tokenSymbol: 'Non-unique symbol of the token.',
   tokenName: 'Full name of the token.',
-  tokenContractId: 'Identifier of a the Smart Contract that implements AEX-9 interface.',
+  tokenContractId: 'Identifier of the Smart Contract that implements AEX-9 interface.',
+  price: 'Fiat price of one token.',
+  marketCap: 'Total fiat market value of all tokens.',
+  totalSupply: 'Amount of tokens minted.',
+  decimals: 'The decimal granularity of each token.',
+  extensions: 'Properties of the tokens.',
+  apiLinks: 'Properties of the tokens.',
+  account: 'Account of the token holder.',
+  amount: 'Amount of the tokens held by its owner.',
+  percentage: 'Percentage of the token held by its owner.',
+  hash: 'The hash of the transaction produced by sending the token.',
```

Comment:
How about replacing "produced" with "created", "dispatched", "committed"? "producing" in context of transaction creation sounds a little exotic to me but that's subjective. Feel free to ignore it

Reply:
It's not always a "send" event so I would suggest to add a generic description, for example:
"Transaction hash of the event".